windows-azure-storage.php

```diff
@@ -214,11 +214,9 @@ function windows_azure_storage_load_textdomain() {
  */
 function windows_azure_plugin_check_prerequisite() {
 	global $wp_version;
-	$php_version = phpversion();
-	$php_compat  = version_compare( $php_version, '8.0.0', '>=' );
-	if ( ! $php_compat ) {
+	if ( ! was_minimum_php_requirement() ) {
```

Comment:
@radeno `was_minimum_php_requirement ` on return php version. It does not compare the required PHP version with the environment PHP version. Can you add PHP version validation?

Reply:
I will suggest to use https://github.com/10up/wp-compat-validation-tool

Reply:
Fixed